02.calendar/cal.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+import dayjs from "dayjs";
+import "dayjs/locale/ja.js";
+import timezone from "dayjs/plugin/timezone.js";
+import utc from "dayjs/plugin/utc.js";
+import minimist from "minimist";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.tz.setDefault("Asia/Tokyo");
+
+function main() {
+  const { m, y } = get_month_year();
+  console.log(`      ${m}月 ${y}`);
+
+  const week_header = ["日", "月", "火", "水", "木", "金", "土"];
+  console.log(week_header.join(" "));
```

Comment:
```suggestion
  console.log("日 月 ... 土");
```

配列に入れるメリットがあまりない気がするんですがどうでしょう？